microsvcs/admin/forum/app/util/handler.js

```diff
@@ -0,0 +1,17 @@
+exports.createErrorResponse = (errormessage) => {
```

Comment:
worth to explore error handling using middleware
https://expressjs.com/en/guide/error-handling.html